src/loki_alert_rules/error_level_logs_found.rule

```diff
@@ -0,0 +1,7 @@
+alert: ErrorLevelLogsFound
+expr: rate({%%juju_topology%%} |= "level=error"[1m]) > 0
```

Comment:
Shall we start using the `job~=.+` thingy instead of `%%juju_topology%%`?

Reply:
Not until we make a decision [on this](https://github.com/canonical/loki-k8s-operator/issues/229).

Reply:
My impression was that it's independent: we don't need `%%juju_topology%%`, and with it it won't pass cos-tool validation, and with `job=~".+"` everything works, so might as well..?